src/cpp/src/visual_language/clip.cpp

```diff
@@ -131,6 +131,83 @@ void bicubic_resize(const clip_image_u8 &img, clip_image_u8 &dst, int target_wid
     }
 }
 
+void bicubic_resize_reorder_normalize(const uint8_t* input, const int nx, const int ny,
+                                      float* target, int target_width, int target_height,
+                                      const float* img_mean3, const float* img_std3) {
+
+    float Cc;
+    float C[5];
+    float d0, d2, d3, a0, a1, a2, a3;
+    int i, j, k, jj;
+    int x, y;
+    float dx, dy;
+    float tx, ty;
+
+    size_t img_size = target_width * target_height;
+    tx = (float)nx / (float)target_width;
+    ty = (float)ny / (float)target_height;
+
+    // Bicubic interpolation; adapted from ViT.cpp, inspired from :
+    //    -> https://github.com/yglukhov/bicubic-interpolation-image-processing/blob/master/libimage.c#L36
+    //    -> https://en.wikipedia.org/wiki/Bicubic_interpolation
+
+    for (i = 0; i < target_height; i++) {
+        for (j = 0; j < target_width; j++) {
+            x = (int)(tx * j);
+            y = (int)(ty * i);
+
+            dx = tx * j - x;
+            dy = ty * i - y;
+
+            float dx2 = dx * dx;
+            float dx3 = dx2 * dx;
+            float dy2 = dy * dy;
```

Comment:
Why don't we use `Interpolate` operation from opset? You can just patch a model with operation.
https://docs.openvino.ai/2025/documentation/openvino-ir-format/operation-sets/operation-specs/image/interpolate-11.html

Reply:
not not much change. You just change ov::Model with a simple patch. Check PR for text-to-image where I patch decoder.
Your manual implementation can slowdown the whole pipeline.